.github/workflows/pull-name.yml

```
@@ -0,0 +1,25 @@
+name: Pull Request Title Format
+
+on:
+  pull_request:
+    branches:
+      - '*'
```

I think you need two stars to get all branches with slashes.
```suggestion
      - '**'
```